### ISSUE TITLE
netflowv9 cflow not working 

### ISSUE THREAD

**Issue by asmeureanu** [secdev/scapy#916]
```
from scapy.all import *
from scapy.contrib import *
from scapy.contrib.bgp import *
from scapy.contrib.ldp import *
from scapy.layers.netflow import *
from scapy.layers.all import *
....
        for packet in packets:
             if (packet.haslayer(NetflowHeaderV9)):
                logger.info(packet.show())
```
No package from the attached pcap is recognized as netflowV9

If I issue ls() i see all the layers/headers/packages available
```
NetflowDataflowsetV9 : Netflow DataFlowSet V9
NetflowFlowsetV9 : Netflow FlowSet V9
NetflowHeader : Netflow Header
NetflowHeaderV1 : Netflow Header v1
NetflowHeaderV5 : Netflow Header v5
NetflowHeaderV9 : Netflow Header V9
NetflowOptionsDataRecordV9 : Netflow Options Data Record V9
NetflowOptionsFlowsetOptionV9 : Netflow Options Template FlowSet V9 - Option
NetflowOptionsFlowsetScopeV9 : Netflow Options Template FlowSet V9 - Scope
NetflowOptionsFlowsetV9 : Netflow Options Template FlowSet V9
NetflowOptionsRecordOptionV9 : Netflow Options Template Record V9 - Option
NetflowOptionsRecordScopeV9 : Netflow Options Template Record V9 - Scope
NetflowRecordV1 : Netflow Record v1
NetflowRecordV5 : Netflow Record v5
NetflowRecordV9 : Netflow DataFlowset Record V9
NetflowTemplateFieldV9 : Netflow Flowset Template Field V9
NetflowTemplateV9 : Netflow Flowset Template V9
```
pcap file 

[capture_flow-active120-inactive60.zip](https://github.com/secdev/scapy/files/1415232/capture_flow-active120-inactive60.zip)

**Comment by gpotter2** [comment]
Hi !
Does https://github.com/secdev/scapy/pull/917 fixes it for you ?

**Comment by asmeureanu** [comment]
Yes, thank you very prompt reply!  
Incredible work and very good timing!

**Comment by vasiqmz** [comment]
@asmeureanu Can you share the code for parsing the CFlow packets via Scapy. I'm having some issues in decoding the flows packet.
Awaiting for your reply.

**Comment by p-l-** [comment]
Fixed in #917.
